internal/mdns/mdns.go

```diff
@@ -135,13 +123,15 @@ func (m *MDNS) start(allowRestart bool) error {
 		LoggerFactory: logging.GetPionDefaultLoggerFactory(),
 	})
 
+	context = context.Interface("mDNSConn", mDNSConn).Err(err)
+
 	if err != nil {
-		scopeLogger.Warn().Err(err).Msg("failed to start mDNS server")
+		context.Err(err).Error().Msg("failed to start mDNS server")
```

Comment:
... and it lead to decisions like this one... where we shouldn't really be adding context to a context unless we've `.With()` it to dupe... so this is technically wrong and will be put back to `logger.Error().Err(err).Msg(` before I'm done...